src/groundStation/uTransceiver.py

```diff
@@ -77,54 +76,57 @@ def listen_zmq(self):
 
         socket.disconnect("tcp://localhost:%s" % self.rxport)
 
+    def handlePipeMode(self):
+        if (time.time() - self.last_tx_time) > self.pipetimeout_s:
+            self.enterPipeMode()
+            return True
```

Comment:
need to add self.last_tx_time = time.time() before this return True

Reply:
Ok